gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -18,6 +18,15 @@
 #  api:
 #    url: http://localhost:8083/management
 
+# Active Pendo Analysis Tool
+# To make our SaaS offering easier to use, we now have the ability to track activity in Pendo. This is disabled by
+# default, but if you'd like to help us in this journey, don't hesitate to reach us to get our key ;)
+#console:
```

Comment:
There is already a `console` key on line 15, so we can merge the analytics part in it?

Reply:
there is another one at 373. that's why I added another one. but if it react you too I will merge the 3